src/components/plots/DoubletScoresPlot.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Vega } from 'react-vega';
+
+import PlatformError from '../PlatformError';
+import { generateSpec, generateData } from '../../utils/plotSpecs/generateDoubletScoresSpec';
+import { loadEmbedding } from '../../redux/actions/embedding';
+import loadCellMeta from '../../redux/actions/cellMeta';
+import { loadCellSets } from '../../redux/actions/cellSets';
+import { loadProcessingSettings } from '../../redux/actions/experimentSettings';
+
+import Loader from '../Loader';
+
+const DoubletScoresPlot = (props) => {
+  const { experimentId, config } = props;
+  const defaultEmbeddingType = 'umap';
+  const plotName = 'doubletScores';
```

Comment:
why don't you use the `plotUuid` that you get passed in `props` for  things requiring plot name instead of `plotName`?

Reply:
plotName is for something different that plotUuid, in this case we are using it for `loadCellMeta` to define which task to pass over to the api (`getDoubletScore` or `getMitochondrialContent`), I think to reflect that better I could rename this variable.

Reply:
apart from this, I believe as a part of [this task](https://biomage.atlassian.net/browse/BIOMAGE-590) we will be refactoring both this and the mitochondrial content plot into one and removing the loading logic from the resulting plot component.

Reply:
sounds good!